ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -242,14 +241,15 @@ open class Tooltip: NSObject, TokenizedControlInternal {
     public var tokenSet: TooltipTokenSet = .init()
     var tokenSetSink: AnyCancellable?
     var fluentTheme: FluentTheme {
-        guard let tooltipView = tooltipViewController?.view else {
+        // Use anchor view to get theme since tooltip view will most likely be nil
```

Comment:
@mischreiber can you confirm if this is an okay workaround? are there any side effects to this/is it hacky? I assumed it'd be okay since the anchor view and tooltip view will always be in the same window, but please let me know if that isn't correct :)

Reply:
this is also concerning that Tooltip.swift holds on the the anchorView and there is additional private let of anchorview inside the TooltipView?